src/Tizen.Wearable.CircularUI.Forms.Renderer/PopupEntryRenderer.cs

```diff
@@ -64,6 +66,30 @@ protected override void OnElementChanged(ElementChangedEventArgs<Xamarin.Forms.E
             }
         }
 
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            base.OnElementPropertyChanged(sender, e);
+            if (_IsPopupOpened && _editor != null)
+            {
+                if (e.PropertyName == XEntry.TextColorProperty.PropertyName)
+                {
+                    _editor.TextColor = Element.TextColor.ToNative();
```

Comment:
How about use Control?
```suggestion
                    _editor.TextColor = Control.TextColor;
```
or TextStyle?
``` c#
    if (e.PropertyName == XEntry.TextColorProperty.PropertyName ||
        e.PropertyName == XEntry.FontSizeProperty.PropertyName ||
        e.PropertyName == XEntry.FontFamilyProperty.PropertyName ||
        e.PropertyName == XEntry.FontAttributesProperty.PropertyName)
    { 
       _editor.TextStyle = Control.TextStyle;
    }
```

Reply:
Done.  set  Control.TextStyle to popup editor's TextStyle.